manifests/crds/network.harvester.cattle.io_nodenetworks.yaml

```diff
@@ -166,8 +166,7 @@ spec:
         type: object
     served: true
     storage: true
-    subresources:
-      status: {}
+    subresources: {}
```

Comment:
Why do we remove the subresource?

Reply:
With subresources, we can not initialize status when creating nodenetwork.